pkg/runtime/pubsub/cloudevents_test.go

```diff
@@ -44,14 +44,14 @@ func TestNewCloudEvent(t *testing.T) {
 			Topic:           "b",
 			Pubsub:          "c",
 			DataContentType: "",
-			TraceID:         "d",
+			TraceParent:     "d",
 		})
 		assert.NoError(t, err)
 		assert.Equal(t, "a", ce["source"].(string))
 		assert.Equal(t, "b", ce["topic"].(string))
 		assert.Empty(t, ce["data"])
 		assert.Equal(t, "text/plain", ce["datacontenttype"].(string))
-		assert.Equal(t, "d", ce["traceid"].(string))
+		assert.Equal(t, "d", ce["traceparent"].(string))
```

Comment:
This is a breaking change.

Reply:
@artursouza it seems like that from this PR's perspective, but the corresponding components-contrib PR retains backward compatibility: https://github.com/dapr/components-contrib/pull/1604.

Reply:
There is a place in this PR that does probably needs to get adjusted for backward compat actually. I'll add a comment.